### PR TITLE
Tweaks to make forking easier

### PR DESCRIPTION
**Pull request by zpalmtree**
Moves hopefully everything needed for forking to the `src/config` folder.

I have not tested the icon files on Windows.

### CODE REVIEW

**Comment by RocksteadyTC** [comment]
got some errors here
```
       "C:\projects\turtlecoin\build\TurtleCoin.sln" (default target) (1) ->
       "C:\projects\turtlecoin\build\src\miner.vcxproj.metaproj" (default target) (9) ->
       "C:\projects\turtlecoin\build\src\miner.vcxproj" (default target) (32) ->
       (ResourceCompile target) -> 
         C:\Libraries\boost_1_67_0\boost/cstdint.hpp(314): error RC2177: constant too big [C:\projects\turtlecoin\build\src\miner.vcxproj]
         C:\Libraries\boost_1_67_0\boost/cstdint.hpp(314): fatal error RC1012: mismatched parenthesis : missing ' [C:\projects\turtlecoin\build\src\miner.vcxproj]
       "C:\projects\turtlecoin\build\TurtleCoin.sln" (default target) (1) ->
       "C:\projects\turtlecoin\build\src\service.vcxproj.metaproj" (default target) (11) ->
       "C:\projects\turtlecoin\build\src\service.vcxproj" (default target) (35) ->
         C:\Libraries\boost_1_67_0\boost/cstdint.hpp(314): error RC2177: constant too big [C:\projects\turtlecoin\build\src\service.vcxproj]
         C:\Libraries\boost_1_67_0\boost/cstdint.hpp(314): fatal error RC1012: mismatched parenthesis : missing ' [C:\projects\turtlecoin\build\src\service.vcxproj]
       "C:\projects\turtlecoin\build\TurtleCoin.sln" (default target) (1) ->
       "C:\projects\turtlecoin\build\src\zedwallet.vcxproj.metaproj" (default target) (12) ->
       "C:\projects\turtlecoin\build\src\zedwallet.vcxproj" (default target) (34) ->
         C:\Libraries\boost_1_67_0\boost/cstdint.hpp(314): error RC2177: constant too big [C:\projects\turtlecoin\build\src\zedwallet.vcxproj]
         C:\Libraries\boost_1_67_0\boost/cstdint.hpp(314): fatal error RC1012: mismatched parenthesis : missing ' [C:\projects\turtlecoin\build\src\zedwallet.vcxproj]
       "C:\projects\turtlecoin\build\TurtleCoin.sln" (default target) (1) ->
       "C:\projects\turtlecoin\build\src\TurtleCoind.vcxproj.metaproj" (default target) (6) ->
       "C:\projects\turtlecoin\build\src\TurtleCoind.vcxproj" (default target) (36) ->
         C:\Libraries\boost_1_67_0\boost/cstdint.hpp(314): error RC2177: constant too big [C:\projects\turtlecoin\build\src\TurtleCoind.vcxproj]
         C:\Libraries\boost_1_67_0\boost/cstdint.hpp(314): fatal error RC1012: mismatched parenthesis : missing ' [C:\projects\turtlecoin\build\src\TurtleCoind.vcxproj]
```

File: src/CryptoNoteCore/Currency.h [hunk 1]
```diff
@@ -10,9 +10,9 @@
 #include <string>
 #include <vector>
 #include <boost/utility.hpp>
-#include "../CryptoNoteConfig.h"
-#include "../crypto/hash.h"
-#include "../Logging/LoggerRef.h"
+#include <config/CryptoNoteConfig.h>
+#include "crypto/hash.h"
```
Comment (brandonlehmann) [inline src/CryptoNoteCore/Currency.h]: Why?

Comment (zpalmtree) [reply src/CryptoNoteCore/Currency.h]: Something wasn't compiling and pointed to this file, hadn't seen any files using `..` to find the header file so I thought I messed up my `sed`

File: src/config/version.h.in [hunk 2]
```diff
@@ -1,7 +1,9 @@
+#include <config/CryptoNoteConfig.h>
+
 #define STR_HELPER(x) #x
 #define STR(x) STR_HELPER(x)
 
-#define PROJECT_NAME "TurtleCoin"
+#define PROJECT_NAME CryptoNote::CRYPTONOTE_NAME
 #define PROJECT_SITE "http://www.turtlecoin.lol"
```
Comment (brandonlehmann) [inline src/config/version.h.in]: Why not move this out too ?

File: src/CryptoNoteCore/Currency.h (hunk 1 above)
Comment (brandonlehmann) [reply src/CryptoNoteCore/Currency.h]: Weird.

File: src/config/version.h.in (hunk 2 above)
Comment (zpalmtree) [reply src/config/version.h.in]: Well, it's in the config folder already, but I guess we could

**Pull request merged**